src/adapter.c

```diff
@@ -442,6 +443,7 @@ int close_adapter(int na)
 		ad->close(ad);
 	//close all streams attached to this adapter
 	//	close_streams_for_adapter (na, -1);
+	ad->is_closing = 1;
```

Comment:
Move this to satipc_close

Reply:
> Move this to satipc_close

Sorry? The function `close_adapter` is called _before_ any other function inside the satipc module. So it's impossible to set the `is_closing` flag before this point. And it's required to set it here as the next lines change the pids, and inside the satipc handling of the pids I need to know if the adapter is closing.